diagnostic.md

```diff
@@ -5,27 +5,35 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    [route] -> [temlate] -> [component1]
+                      '---> [component2]
+
+    A list of lists where each list of lists is also a list could be modeled by
+    a list of lists, where each list is a component.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    - directory inside the app/application/components
+    - template.hbs inside directory
+    - component.js file inside directory
+    - test file in tests/integration/components/example-list/component-test.js
+
```

Comment:
Template files (`template.hbs`) are responsible for containing markup
(defining structure), binding data, and binding and dispatching actions.

Components objects (`component.js`) are responsible for defining actions,
declaring HTML attributes, and other presentation or behavior.

In general, templates are responsible for markup, while components are
responsible for presentation and behavior.

.5/1